### PR TITLE
x11-misc/xembedsniproxy: version bump to 5.12.0

### PR DESCRIPTION
**Pull request by Hummer12007**

### CODE REVIEW

File: x11-misc/xembedsniproxy/xembedsniproxy-5.12.0.ebuild [hunk 1]
```diff
@@ -6,7 +6,7 @@ EAPI=6
 inherit cmake-utils
 
 DESCRIPTION="A program to use legacy Xembed tray icons with SNI-only trays"
-HOMEPAGE="https://kde.org"
+HOMEPAGE="https://github.com/KDE/plasma-workspace/tree/master/xembed-sni-proxy"
```
Comment (a17r) [inline x11-misc/xembedsniproxy/xembedsniproxy-5.12.0.ebuild]: That's a git subdir on a mirror of the project, do you think that's appropriate as homepage?

Comment (Hummer12007) [reply x11-misc/xembedsniproxy/xembedsniproxy-5.12.0.ebuild]: Hm, you're right about it being a mirror, thanks. Though unfortunately, no other page (I found) refers to the program, and I was convinced in IRC that a git repo subdir IS more appropriate than kde.org.

**Comment by Hummer12007** [comment]
Comments addressed.

File: x11-misc/xembedsniproxy/xembedsniproxy-5.12.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2017 Gentoo Foundation
```
Comment (mgorny) [inline x11-misc/xembedsniproxy/xembedsniproxy-5.12.0.ebuild]: Your calendar seems to be late, Sir.

**Comment by mgorny** [comment]
I'm going to fix that date and merge it, provided that it builds fine.

File: x11-misc/xembedsniproxy/xembedsniproxy-5.12.0.ebuild [hunk 3]
```diff
@@ -6,7 +6,7 @@ EAPI=6
 inherit cmake-utils
 
 DESCRIPTION="A program to use legacy Xembed tray icons with SNI-only trays"
-HOMEPAGE="https://kde.org"
+HOMEPAGE="https://cgit.kde.org/plasma-workspace.git/tree/xembed-sni-proxy/Readme.md"
```
Comment (mgorny) [inline x11-misc/xembedsniproxy/xembedsniproxy-5.12.0.ebuild]: This doesn't belong in this commit.

Comment (Hummer12007) [reply x11-misc/xembedsniproxy/xembedsniproxy-5.12.0.ebuild]: Fixed

**Comment by mgorny** [comment]
Thanks a lot. I'll squash that `1` and merge it ;-).

**Comment by Hummer12007** [comment]
Err, another rebase fucked up. Sorry.
